.gitignore

```diff
@@ -0,0 +1 @@
+resources/dbcreds.json
```

Comment:
Since we are ignoring this file. Can we mention this in the documentation. An example of how and where this file should be created for setting up database.

Reply:
Sure I'll open an issue for it.